spec/features/item_registration_spec.rb

```diff
@@ -72,7 +72,7 @@
       'workflow_id' => 'goobiWF',
       'label' => 'object title',
       'tag' => ['Process : Content Type : Book (ltr)', 'tag : test', "Registered By : #{user.sunetid}"],
-      'rights' => 'default',
+      'rights' => 'world',
```

Comment:
@sul-dlss/infrastructure-team any ideas why I'd need to make this change to fix the build? FWIW, I was seeing this in `main` as well prior to switching into this branch. Did a default change? Or does Argo need more cocina-related changes? cc: @jcoyne @justinlittman